app/javascript/app-styles.sass

```diff
@@ -0,0 +1,3 @@
+@import '../assets/stylesheets/app'
```

Comment:
I couldn't find an `app/assets/stylesheets/app.(css|sass)`, should it have been committed?